airflow/www/utils.py

```diff
@@ -351,9 +355,8 @@ def get_attr_renderer():
         'doc_yaml': lambda x: render(x, lexers.YamlLexer),
         'doc_md': wrapped_markdown,
         'python_callable': lambda x: render(
-            inspect.getsource(x), lexers.PythonLexer),
+            inspect.getsource(x) if x is not None else None, lexers.PythonLexer),
```

Comment:
should this line be `inspect.getsource(x) if x is not None else ''`? For your .jira description, the issue happens when None passes to render function which causes exception?

Reply:
The behavior is correct either way: `PythonLexer` can handle `None`, but `inspect.getsource`. can't. The unit tests make sure that this is the case.